arangod/Pregel/Algorithm.h

```diff
@@ -110,17 +112,16 @@ struct Algorithm : IAlgorithm {
   virtual vertex_compensation* createCompensation(WorkerConfig const*) const {
     return nullptr;
   }
-  virtual std::set<std::string> initialActiveSet() {
-    return std::set<std::string>();
-  }
+  virtual std::set<std::string> initialActiveSet() { return {}; }
 
-  virtual uint32_t messageBatchSize(WorkerConfig const& config,
-                                    MessageStats const& stats) const {
+  [[nodiscard]] virtual uint32_t messageBatchSize(
+      WorkerConfig const& config, MessageStats const& stats) const {
     if (config.localSuperstep() == 0) {
       return 500;
     } else {
-      double msgsPerSec = stats.sendCount / stats.superstepRuntimeSecs;
-      msgsPerSec /= config.parallelism();  // per thread
+      double msgsPerSec =
+          static_cast<double>(stats.sendCount) / stats.superstepRuntimeSecs;
+      msgsPerSec /= static_cast<double>(config.parallelism());  // per thread
```

Comment:
I see your point and I agree. On the other hand, (1) it will realistically never be done in another way, (2) it distracts because my IDE marks such places, (3) it is relatively easy to review if such changes are not too numerous in one PR and do not interleave with the actual changes (a lot).